src/ripple/rpc/tests/Subscribe.test.cpp

```diff
@@ -249,10 +249,28 @@ class Subscribe_test : public beast::unit_test::suite
         expect(jv[jss::status] == "success");
     }
 
+    static
+    std::unique_ptr<Config>
+    makeValidatorConfig()
+    {
+        auto p = std::make_unique<Config>();
+        setupConfigForUnitTests(*p);
+
+        // If the config has valid validation keys then we run as a validator.
+        auto const seed = parseBase58<Seed>("shUwVw52ofnCUX5m7kPTKzJdr4HEH");
+        if (!seed)
+            Throw<std::runtime_error> ("Invalid seed specified");
+        p->VALIDATION_PRIV = generateSecretKey (KeyType::secp256k1, *seed);
+        p->VALIDATION_PUB =
```

Comment:
This is actually problematic as is: it relies on the fact that the existing configuration loading don't touch the `VALIDATION_PUB` and `VALIDATION_PRIV` variables if a `[node_seed]` isn't specified. It's also brittle because we now add a "hidden" dependency between two different string literals - they must be paired and if one is changed, so must the other.

The "canonical" way to do this is to _simply_ have:

    p->NODE_SEED = "shUwVw52ofnCUX5m7kPTKzJdr4HEH";

As @vinniefalco points out, this will actually incur an expensive `secp256k1` operation every time this function is invoked - but this only happens once per unit test so I'm not _too_ concerned. 

Still, if you want to pico-optimize the unit tests (for reference the `Stoppable` shutdown takes almost three longer) try:

    p->VALIDATION_PRIV = parseBase58<SecretKey>("paEdUCVVCNnv4aYBepid9Xh3NaAr9xWRw2vh351piFJrxQwvExd");
    p->VALIDATION_PUB = parseBase58<PublicKey("n9MvFGjgv1kYkm7bLbb2QUwSqgzrQkYMYHXtrzN8W28Jfp2mVihq");